Add coverage for MSTG-CODE-9 (#2089)
OWASP/owasp-mstg: Document/0x06b-Basic-Security-Testing.md

- [Swift Packages](https://developer.apple.com/documentation/swift_packages): Xcode 11 add supports for Swift packages, which are reusable components of Swift, Objective-C, Objective-C++, C, or C++ code that developers can use in their projects and are distributed as source code. Since Xcode 12 they can also [bundle resources](https://developer.apple.com/videos/play/wwdc2020/10169/), such as images, storyboards, and other files. Since Package libraries are [static by default](https://developer.apple.com/videos/play/wwdc2019/408/?time=739). Xcode compiles them, and the packages they depend on, and then links and combines everything into the application.

> You can easily visualize native libraries from Passionfruit by clicking on "Modules":

Reviewer: ```suggestion
You can visualize native libraries via Passionfruit by clicking on "Modules":
```
